style.css

```diff
@@ -5,10 +5,170 @@
     src: url("./Assets/fonts/github/Inter-VariableFont_slnt\,wght.ttf");
   }
 
-  .Navmenu {
+  .Navmenux {
     display: none;
   }
 
+  .imagedescription {
+    width: 100%;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  .popout {
+    height: 90%;
+    border-radius: 5px;
+    width: 90%;
+    background-color: white;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 70px;
+    margin-bottom: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .popout-container {
+    width: 100%;
+    height: 100%;
+    position: fixed;
+    background-color: rgba(33, 32, 32, 0.4);
+    -webkit-backdrop-filter: blur(5px);
+    backdrop-filter: blur(5px);
+    z-index: 999;
+    display: flex;
+  }
```

Comment:
kindly make sure that when the pop-up appears, the content of the background is blurred for both desktop and mobile versions to match the figma design as shown below

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-14 11-24-27](https://user-images.githubusercontent.com/65233860/195838651-c7bf7bff-3985-41ed-b204-1947109facbc.png)  |  ![Screenshot from 2022-10-14 11-37-26](https://user-images.githubusercontent.com/65233860/195838686-2dc4fc01-efdb-47aa-9b22-826ce361e5ef.png)